packages/react-vapor/src/components/slider/SliderUtils.ts

```diff
@@ -15,55 +17,48 @@ export const getValuesPositionOnRange = (value: number[], crossingPoint: number)
     return null;
 };
 
-export const getCrossingPoint = (min: number, max: number): number => Math.round(((0 - min) / (max - min)) * 100);
-
-export const convertInitialValuetoRangeValue = (min: number, max: number, initialValue: number): number =>
-    Math.round(((initialValue - min) / (max - min)) * 100);
-
 export const handleIsAtCrossingPoint = (
-    lowRange: number,
-    highRange: number,
+    leftHandlePosition: number,
+    rightHandlePosition: number,
     handleIndex: number,
     crossingPoint: number
 ) =>
-    (handleIndex === 1 && lowRange < crossingPoint && highRange === crossingPoint) ||
-    (handleIndex === 0 && highRange > crossingPoint && lowRange === crossingPoint);
+    (handleIndex === 1 && leftHandlePosition < crossingPoint && rightHandlePosition === crossingPoint) ||
+    (handleIndex === 0 && rightHandlePosition > crossingPoint && leftHandlePosition === crossingPoint);
 
-export const getComputedRangeValue = (
-    lowRange: number,
-    highRange: number,
-    min: number,
-    max: number,
+export const getOutputValue = (
+    leftHandlePosition: number,
+    rightHandlePosition: number,
     crossingPoint: number
 ): number => {
-    if (lowRange < crossingPoint) {
-        return Math.round(((crossingPoint - lowRange) * min) / crossingPoint);
-    } else if (highRange > crossingPoint) {
-        return Math.round(((highRange - crossingPoint) * max) / (100 - crossingPoint));
+    if (leftHandlePosition < crossingPoint) {
+        return leftHandlePosition;
```

Comment:
Why remove the Math.round out of interest?

Reply:
Its not about the `Math.round` as much as removing all the calculations. By setting a `min` and a `max` directly on the `Range` component, it scales itself so don't have any scaling calculations to do anymore.

Reply:
ohh that makes more sense